### PR TITLE
Update getInstalledRelatedApps() for Windows support.

### PR DESCRIPTION
**Pull request by jpmedley**
@petele I've used the `scheduled` tag so that we don't need to remember to push this at the end of next month.

### CODE REVIEW

File: src/site/content/en/blog/get-installed-related-apps/index.md [hunk 1]
```diff
@@ -59,13 +65,14 @@ iterated on the design.
 | 2. Create initial draft of specification     | [Complete][spec]             |
 | 3. Gather feedback and iterate on design     | Complete                     |
 | 4. Origin trial                              | Complete                     |
-| 5. **Launch**                                | Chrome 80, and beyond        |
+| 5. **Launch**                                | **Android: Chrome 80<br/>Edge for Windows: Chrome 85** |
```
Comment (kaycebasques) [inline src/site/content/en/blog/get-installed-related-apps/index.md:68]: @petele & @jpmedley, just checking: Edge wants to be known as "Edge for Windows: Chrome 85"?

File: src/site/content/en/blog/get-installed-related-apps/index.md [hunk 2]
```diff
@@ -228,8 +232,8 @@ relatedApps.forEach((app) => {
 {% Aside %}
 `getInstalledRelatedApps()` is currently supported on the following platforms:
 
-* Android: Chrome 80+
-* Windows: Edge (coming soon), Chrome (coming soon)
+* Android: Chrome 80 and later
+* Windows: Edge Chrome 85 and later; Chrome (coming soon)
```
Comment (kaycebasques) [inline src/site/content/en/blog/get-installed-related-apps/index.md:236]: @petele & @jpmedley another instance of mixing Edge and Chrome here (Edge Chrome 85)

File: src/site/content/en/blog/get-installed-related-apps/index.md (hunk 1 above)
Comment (kaycebasques) [reply src/site/content/en/blog/get-installed-related-apps/index.md:68]: To me it seems we should just refer to it as Edge 85. The fact that it's using Chromium underneath the hood is an implementation detail. I don't recall us using this phrasing for any of the other Chromium-based browsers. E.g. we don't say "Samsung Internet: Chrome 84"

Comment (jpmedley) [reply src/site/content/en/blog/get-installed-related-apps/index.md:68]: I wasn't thinking clearly. I had my week off on my mind.

**Comment by jpmedley** [comment]
@kaycebasques PTAL

**Comment by jpmedley** [comment]
DO NOT DELETE THIS BRANCH. It was merged prematurely. We need the branch to reopen the PR for real in about two weeks.

@kaycebasques @robdodson

**Comment by robdodson** [comment]
I was out on vacation when this PR was worked on so I missed these notifications.

Is there anything you'd like me to do on my end or are we just waiting to reopen the branch PR?

**Comment by jpmedley** [comment]
@robdodson I opened a new PR in another branch. There's nothing to do here.

**Pull request merged**